azure-iot-device/samples/pnp/README.md

```diff
@@ -1,14 +1,38 @@
-# Samples to demonstrate Azure IoT Plug and Play
+---
+page_type: sample
+description: "A set of Python samples that show how a device that uses the IoT Plug and Play conventions interacts with either IoT Hub or IoT Central."
+languages:
+- python
+products:
+- azure-iot-hub
```

Comment:
This could be confusing since they are formatted as package names. `azure-iot-hub` is a real package in Python, the other two are not. I assume however, this is not referring to specific packages?